tests/test-deploy-ad-server.sh

```diff
@@ -49,7 +49,12 @@ while [ $tries -lt 120 ] && [ $rc -ne 0 ]; do
 	echo -n "."
 done
 echo
-[ $rc -eq 0 ] || _error "Error: samba ad did not become reachable"
+if [ $rc -ne 0 ] ; then
+	echo -n "---- DEBUG -----"
+	kubectl exec "${podname}" -- smbclient -d5 -N -L 127.0.0.1
+	echo -n "---- END DEBUG -----"
+	_error "Error: samba ad did not become reachable"
+fi
```

Comment:
I hope this additional change is not intended with this PR. If yes, are you planning to make such a change in _tests/test-deploy-ad-member.sh_?

Reply:
It's a hack. When I run the script locally it passes. I have a personal convention of putting "xxx" in the commit message if its temporary/incomplete like some people put "wip".

Reply:
I guessed right.